index.html

```diff
@@ -25,70 +25,310 @@
+            <section>
+              <h4 class="subhead">CANOPY</h4>
+              <img src="images/Counter.svg" alt="counter shape"><h6 class="subhead">Front End Dev</h6>
+              <img src="images/Counter.svg" alt="counter shape"><h6 class="subhead">2022</h6>
+            </section>
+            <section class="pageone-text">
+              <div>daily selection of privately personalized reads; no accounts or sign-ups required.</div>
+            </section>
+            <section class="buttons-section">
+              <ul >
+                <li class="js-css-html-button-ul">
+                  <button type="button" class="html-css-js-button">html</button>
+                </li>
+                <li class="js-css-html-button-ul">
+                  <button type="button" class="html-css-js-button">css</button>
+                </li>
+                <li class="js-css-html-button-ul">
+                  <button type="button" class="html-css-js-button">javascript</button>
+                </li>
+              </ul>
+            </section>
+            <section>
+              <button type="button" class="footer-button">See Project</button>
+            </section>
+        </section>
+      </div>
+      <div class="portfolio-container page-four">
+        <header class="pageone-header">
+          <img src="images/SnapshootPortfolio4.svg" alt="A snapshoot Portfolio Header From Figma">
+        </header>
+        <section class="pageone-body">
+          
+            <h2 class="page-two-text"><strong>Multi-Post</strong></h2>
+            <h2 class="page-two-text"><strong>Stories</strong></h2>
+          
+            <section>
+              <h4 class="subhead">CANOPY</h4>
+              <img src="images/Counter.svg" alt="counter shape"><h6 class="subhead">Front End Dev</h6>
+              <img src="images/Counter.svg" alt="counter shape"><h6 class="subhead">2022</h6>
+            </section>
+            <section class="pageone-text">
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </section>
+            <section class="buttons-section">
+              <ul >
+                <li class="js-css-html-button-ul">
+                  <button type="button" class="html-css-js-button">html</button>
+                </li>
+                <li class="js-css-html-button-ul">
+                  <button type="button" class="html-css-js-button">css</button>
+                </li>
+                <li class="js-css-html-button-ul">
+                  <button type="button" class="html-css-js-button">javascript</button>
+                </li>
+              </ul>
+            </section>
+            <section>
+              <button type="button" class="footer-button page-two-footer">See Project</button>
+            </section>
+        </section>
+      </div>
+      <div class="portfolio-container page-five-div">
+        <header class="text about-myself">
```

Comment:
- [ ] Kindly make sure to wrap this work section into the `section` tag which will increase your work looks more semantically professional :wink: